webots_ros2_turtlebot/launch/robot_launch.py

```diff
@@ -17,35 +17,70 @@
 """Launch Webots TurtleBot3 Burger driver."""
 
 import os
+import pathlib
 from launch.substitutions import LaunchConfiguration
 from launch.actions import DeclareLaunchArgument
 from launch.substitutions.path_join_substitution import PathJoinSubstitution
-from launch.actions import IncludeLaunchDescription
-from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch import LaunchDescription
+from launch_ros.actions import Node
+import launch
 from ament_index_python.packages import get_package_share_directory
+from webots_ros2_core.webots_launcher import WebotsLauncher
```

Comment:
Sorry for the review in hindsight. But this also introduces a dependency on `webots_ros2_core`